### ISSUE TITLE
508: Search box label

### ISSUE THREAD

**Issue by leahbannon** [18F/fbopen#94]
1.2.2A - The main search box is missing its label. A label with a "for" tag would solve this issue. You could also give the box a title tag, but this is less commonly read by screen readers but is still acceptable.
